src/backend/base/langflow/api/utils.py

```diff
@@ -134,12 +134,12 @@ def format_elapsed_time(elapsed_time: float) -> str:
 async def build_graph_from_data(flow_id: str, payload: dict, **kwargs):
     """Build and cache the graph."""
     graph = Graph.from_payload(payload, flow_id, **kwargs)
-    for vertex_id in graph._has_session_id_vertices:
+    for vertex_id in graph.has_session_id_vertices:
```

Comment:
You probably need to update the method `define_vertices_lists` because of this

Reply:
done